charts/oes/charts/spinnaker/templates/configmap/halyard-init-script.yaml

```diff
@@ -47,7 +47,9 @@ data:
     {{- else }}
     #!/bin/bash -x
     rm -rf /tmp/spinnaker/.hal
-    git clone $GIT_CLONE_PARAM /tmp/spinnaker/.hal
+    git clone $GIT_CLONE_PARAM /tmp/spinnaker/test
+    cp -pr /tmp/spinnaker/test/{{ .Values.gitopsHalyard.repo.halConfigPath }} /tmp/spinnaker/.hal
```

Comment:
Why do we need this intermediate step here?

Reply:
We are implementing the folder structure here,to handle all the three gitops repo, dynamic accounts, pipeline promotion in one single git repo.